src/main/java/com/hpe/adm/octane/ideplugins/services/ui/FormLayout.java

```diff
@@ -65,11 +67,16 @@ public void setFormLayoutSections(List<FormLayoutSection> formLayoutSections) {
         this.formLayoutSections = formLayoutSections;
     }
 
-    public boolean isDefault() {
+    public String isDefault() {
```

Comment:
this is the naming convention for booleans; since you changed the type, you also need to change the name